host-interaction/process/dump/create-process-memory-minidump.yml

```diff
@@ -5,6 +5,8 @@ rule:
     namespace: host-interaction/process/dump
     author: michael.hunhoff@mandiant.com
     scope: basic block
+    mbc:
+      - Process::Create Process [C0017]
```

Comment:
MiniDumpWriteDump writes to a file instead of creating a process. I suggest to change this.

Reply:
Changed to Write File